npm/private/npm_import.bzl

```diff
@@ -667,10 +667,7 @@ def _npm_import_links_rule_impl(rctx):
                 deps[dep_store_target].append(dep_name)
     else:
         for (dep_name, dep_version) in rctx.attr.deps.items():
-            if dep_version.startswith("link:") or dep_version.startswith("file:"):
-                dep_store_target = """"//{root_package}:{package_store_root}/{{}}/{package_store_name}".format(link_root_name)"""
-            else:
-                dep_store_target = """":{package_store_root}/{{}}/{package_store_name}".format(link_root_name)"""
+            dep_store_target = """":{package_store_root}/{{}}/{package_store_name}".format(link_root_name)"""
```

Comment:
Merge with the next line and drop the roor_package substitution unless I'm missing something?

Reply:
You're correct, unless I'm missing something 👍